src/styles/stars.scss

```diff
@@ -0,0 +1,21 @@
+.stars {
+  display: flex;
+  margin: 0;
+  &__star {
+    background-image: url(../images/star.svg);
+    background-size: 80%;
```

Comment:
The background-size property is set to 80%, which might not be following the design requirements. If the design specifies that the stars should be a certain size, it would be better to use a fixed size in pixels or another unit that matches the design specification.